docs/ReleaseNotes.Nautilus.November.2017.md

```diff
@@ -0,0 +1,22 @@
+# CM-Well Version Release Notes - Nautilus (November 2017) #
+
+## Change Summary ##
+
+
+ Title | Git Issue | Description 
+:------|:----------|:------------
+Play minor version upgrade | [287](https://github.com/thomsonreuters/CM-Well/pull/287) | Play 3rd-party package upgraded to version 2.6.7
+Integration test improvements | [275](https://github.com/thomsonreuters/CM-Well/pull/275) | Integration test improvements
+**qstream** now supports **length** parameter | [285](https://github.com/thomsonreuters/CM-Well/pull/285) | The **qstream** streaming operator now supports the **length** parameter, which is a hint as to the maximum number of infotons you want to receive.
```

Comment:
> "which is a **hint** as to the "

not a "hint", it's a hard limit.
If stream contain more infotons, it is being cut off amid after `$length` infotons has been emitted.

Reply:
Fixed.